ui/server/src/main/scala/pl/touk/nussknacker/ui/config/processtoolbar/ToolbarButtonConfig.scala

```diff
@@ -0,0 +1,71 @@
+package pl.touk.nussknacker.ui.config.processtoolbar
+
+import io.circe.{Decoder, Encoder}
+import pl.touk.nussknacker.ui.config.processtoolbar.ToolbarButtonConfigType.ToolbarButtonType
+
+case class ToolbarButtonConfig(
+  `type`: ToolbarButtonType,
+  title: Option[String],
+  icon: Option[String],
+  templateHref: Option[String],
+  hide: Option[ToolbarCondition],
+  disabled: Option[ToolbarCondition]
+) {
+
+  if (ToolbarButtonConfigType.requiresHrefTemplate(`type`)) {
+    require(templateHref.isDefined, s"Button ${`type`} requires param: 'templateHref'.")
```

Comment:
if !requiresHrefTemplate we and template isDefined we should also throw exception?

Reply:
Hmm, maybe.. ;)

Reply:
Added